android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIAbstractTabGroup.java

```diff
@@ -126,7 +126,7 @@
 	public abstract String getTabTitle(int index);
 
 	// region protected fields
-	protected static final String TAG = "TiUITabLayoutTabGroup";
+	protected static String TAG = "";
```

Comment:
I don't like us having a settable static TAG variable here. If one TabGroup style ever happened to launch another TabGroup child activity, then the last one would clobber first's TAG name.

Personally, I'm okay with `TAG` being set to "TiUIAbstractTabGroup" from a traceability standpoint. We always set TAG to the Java class name and not the JS type name anyways.